spec/models/user_spec.rb

```diff
@@ -0,0 +1,11 @@
+# spec/models/user_spec.rb
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it 'is valid with a name and email' do
+    # user = User.new(name: "Example User", email: "user@example.com")
+    user = User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+    expect(user).to be_valid
+  end
```

Comment:
- [ ] From the project requirements, you are supposed to implement Model specs has Validation tests which looks like the followin: 
```ruby
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```
Please add them to your unit tests